pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/SomaticResultUpload.java

```diff
@@ -312,6 +319,23 @@ private static SomaticResultUpload getResult(ResultSet rs) throws SQLException {
             rs.getLong(DBConstants.CREATED_DATE));
     }
 
+    /**
+     * Only use this for testing.  Hard deletes the row.  For production code,
+     * we only soft delete, so please use other delete methods for normal operations.
+     */
+    @VisibleForTesting
+    public static void hardDeleteSomaticDocumentById(Handle handle, long id) {
```

Comment:
Sad that the currency in the code is long, since the ID is actually an int in the DB.
Not worth fixing at this point.